knowledge-base-public/src/components/knowledge-base/KnowledgeBasePublicSection.js

```diff
@@ -85,6 +116,17 @@ const KnowledgeBasePublicSection = ({ item, isRoot, isLoading, device }) => {
             Rechercher un article
           </button>
         </div>
+        <div className="flex justify-center bg-[#32257F]">
+          <div className="mx-auto mt-4 flex max-w-[90%] lg:max-w-[70%] flex-row flex-wrap justify-center">
+            {top5Article.map(({ _id, slug, text }) => (
+              <div key={_id} className="m-1 w-full lg:w-auto rounded-2xl bg-blue-100 px-2 py-1 text-center">
+                <Link href={`/base-de-connaissance/${slug}`} aria-label={`Lire l'article ${text}`} alt={`Lire l'article ${text}`}>
+                  <h3 className="text-sm font-medium leading-5 text-blue-800">{text}</h3>
+                </Link>
+              </div>
+            ))}
+          </div>
+        </div>
```

Comment:
C'est un peu compliqué je trouve, pourquoi il y a une div dans une div et un h3 dans un Link dans une autre div ?
Proposition avec seulement deux balises :
```suggestion
          <div className="mx-auto mt-4 flex max-w-[90%] lg:max-w-[70%] flex-wrap justify-center gap-2">
            {top5Article.map(({ _id, slug, text }) => (
              <Link
                key={_id}
                className="rounded-full bg-blue-100 px-4 py-1 text-center text-sm font-medium leading-5 text-blue-800"
                href={`/base-de-connaissance/${slug}`}
                aria-label={`Lire l'article ${text}`}
              >
                {text}
              </Link>
            ))}
          </div>
```